docs/releases/2.7.0.rst

```diff
@@ -0,0 +1,235 @@
+    they'll be synchronized based on the `last_sync_revision` field of the
+    store.
+  - Tests have been resurrected.
+  - Upgraded jQuery to 2.x and applied a bunch of fixes to the Tipsy plugin,
+    avoiding ad-hoc hacks to remove dangling tips.
+  - Translation projects now have a ``creation_time`` field.
+  - Dropped code for several external apps from Pootle codebase. Also upgraded
+    to newer versions of those apps.
+  - The default caching in database has been replaced by local-memory caching.
+    Using memcached is still highly recommended.
+  - Fixed and avoided any inconsistencies in the unit's submitter information.
+
+
+- Sysadmins:
+
+  - The registration and authentication layer is now handled by django-allauth.
+    This gives Pootle implicit support for OpenID, OAuth, OAuth2 and Persona
+    sign-in protocols.
+  - Integrated ElasticSearch-based local TM server into Pootle. Unit
+    submissions update the index instantly.
+  - Upgrade: Direct upgrade is now only possible from 2.6.0
+  - The report target for reporting string errors has been dropped in favor of
+    a report email address. The report is now sent using an automatically
+    pre-filled  contact form. If the project doesn't have a report email then
+    the reports are sent to :setting:`POOTLE_REPORT_STRING_ERRORS_EMAIL`.
+
+
+- Administrators:
+
+  - Only admins can set the fuzzy flag on the unit. Non-admin users are not
+    shown the fuzzy checkbox by default, but in case it's displayed (because
+    the unit was already fuzzy, or some other action turned it fuzzy, such as
+    using MT), they are always forced to clear the flag.
+  - Projects can be disabled from the administration page, allowing to hide
+    them temporarily while retaining statistics.
+  - The ``view`` permission is now project-centric, it can be set server-wide
+    or for projects, but not for individual language-project combinations.
+
+
+
+Removed features
+================
+
+We have dropped some legacy features we kept around for allowing a swift
+upgrade in the past.
+
+The removal of some of these feature required extensive changes on the
+upgrading code, which resulted in that upgrading directly from ancient versions
+is no longer possible. In case you are trying to upgrade from Pootle 2.1.0 or
+older you must first upgrade to 2.1.6 before upgrading to this version.
+
+These are the features we got rid of:
+
+- .pending and .tm files support: Not necessary since the updatetm tool was
+  removed in Pootle 2.5.0.
+- Live translation: Rarely enabled, and it was discouraged to use it.
+- Autosync: It was recommended to never use it. The files can now be synced
+  instead using the appropriate management commands.
+
+The voting feature for terminology suggestions has also been removed, due to
+its low popularity and high maintenance cost. In order to save existing voting
+data, make a backup of the ``voting_vote`` table before upgrading.
```

Comment:
I would remove the advice to backup the table, since in the upgrade instructions the backup of the whole database is explicitly warned.